src/util/imagefiledata.cpp

```diff
@@ -0,0 +1,26 @@
+#include "util/imagefiledata.h"
+
+#include <QByteArray>
+#include <QDebug>
+#include <QFile>
+#include <QFileInfo>
+#include <QImage>
+
+ImageFileData::ImageFileData(const QString& coverArtFilePath)
```

Comment:
This function can become a static ImageFileData::FromFilePath() function that calls that the `ImageFileData::ImageFileData(const QByteArray& coverArtBytes)`
to avoid to read the file twice.

Reply:
I am just for asking for to learn. Sorry if that is too obvious, ı just couldn't get it. How the file can be read twice? By override?